src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import { render, screen } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  render(<App />);
+  const linkElement = screen.getByText(/learn react/i);
+  expect(linkElement).toBeInTheDocument();
+});
```

Comment:
- [ ] I think it would be better also to fix the issue in the test as some tests are failing kindly fix this to complete the project requirements and to ensure that the project works as expected 👍 

<img width="1152" alt="Screen Shot 2023-10-13 at 19 16 38" src="https://github.com/gdvl777/metrics/assets/102757126/cac4c63c-3bdf-43ac-ac9b-fb02f5c83eed">